.github/workflows/list_components.yml

```diff
@@ -92,9 +128,65 @@ jobs:
         shell: bash
     steps:
     - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
     - name: Intel® oneAPI Base Toolkit
-      run: scripts/list_components_macos.sh $MACOS_BASEKIT_URL
+      run: scripts/list_components_macos.sh $MACOS_BASEKIT_URL ${{ github.workflow }}_${{ github.job }}_basekit
     - name: Intel® oneAPI HPC Toolkit
-      run: scripts/list_components_macos.sh $MACOS_HPCKIT_URL
+      run: scripts/list_components_macos.sh $MACOS_HPCKIT_URL ${{ github.workflow }}_${{ github.job }}_hpckit
     - name: Intel® oneAPI Rendering Toolkit
-      run: scripts/list_components_macos.sh $MACOS_RENDERKIT_URL
+      run: scripts/list_components_macos.sh $MACOS_RENDERKIT_URL ${{ github.workflow }}_${{ github.job }}_renderkit
+
+    - name: Upload artifacts
+      uses: actions/upload-artifact@v2
+      with:
+        name: ${{ github.workflow }}_${{ github.job }}
+        path: ${{ github.workflow }}_${{ github.job }}*
+
+  publish:
+    runs-on: ubuntu-20.04
+    needs: [windows, linux, linux_apt, linux_yum_dnf, macos]
+    defaults:
+      run:
+        shell: bash
+        working-directory: doc
+    steps:
+    - uses: actions/checkout@v2
+    - name: Download artifacts
+      uses: actions/download-artifact@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install prerequisites
+      run: python -m pip install --quiet -r requirements.txt
+    - name: Build doc
+      run: |
+        cp ../${{ github.workflow }}_*/* source
+        find source -type f -maxdepth 1 ! -name "*apt*" ! -name "*yum*" -exec sed -i -e 1,5d {} \;
+        find source -type f -maxdepth 1 -name "*windows*" -exec sed -i -e 1,2d {} \;
```

Comment:
This removes unnecessary lines from the output of list_components jobs.
The format of component list on GitHub pages probably needs to be consistent across OS and dist channels, which would require more post-processing of this data. Such post-processing will be added once the desired format is identified.

Reply:
The doc looks nice. I think you can leave it as-is.

If in some future point we wanted to make the formatting more consistent, then I would convert the output of the list components script to csv, and then use the sphinx csv table to format everything into tables.